libs/ml/include/ml/optimisation/adagrad_optimiser.hpp

```diff
@@ -17,14 +17,9 @@
 //
 //------------------------------------------------------------------------------
 
-#include "math/standard_functions/sqrt.hpp"
 #include "ml/core/graph.hpp"
 #include "ml/optimisation/optimiser.hpp"
 
-#include <memory>
-#include <string>
-#include <vector>
```

Comment:
I would suggest to leave that. It seems to me more consistent along with usage of mentioned types in function declarations. But I'm not insist